client/src/components/Map/LeafletMap.js

```diff
@@ -1,33 +1,48 @@
 import React from "react";
-import { MapContainer, TileLayer, Popup } from "react-leaflet";
+import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
 import styled from "styled-components";
 import "leaflet/dist/leaflet.css";
-import { ReactComponent as MarkerIcon } from "../../assets/marker.svg";
-import Marker from "react-leaflet-enhanced-marker";
 
+// import Marker from "react-leaflet-enhanced-marker";
 import PropTypes from "prop-types";
+import L from "leaflet";
```

Comment:
Import L does not say that much what you are importing, in case you can change the importing name to something more describing, that could be a good idea. :)

Reply:
This is s the default Marker import. I wasn't able to display the markers before I imported this  so I probably won't change this